src/applicationcomposer/messageHandlers/deployMessageHandler.ts

```diff
@@ -3,10 +3,49 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import vscode from 'vscode'
+import vscode, { Uri } from 'vscode'
 import { Command, DeployRequestMessage, DeployResponseMessage, MessageType, WebviewContext } from '../types'
 import { SamSyncResult } from '../../shared/sam/sync'
 import { telemetry } from '../../shared/telemetry/telemetry'
+import { Auth } from '../../auth/auth'
+import { promptAndUseConnection } from '../../auth/utils'
+import { openUrl } from '../../shared/utilities/vsCodeUtils'
+import { localize } from 'vscode-nls'
+import { StatefulConnection } from '../../auth/connection'
+
+const credentialsDocLink = 'https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-credentials.html'
```

Comment:
any reason not to re-use this constant? https://github.com/aws/aws-toolkit-vscode/blob/2a1b31cf3218c3eea6fa56d69a0fbd7a894eaad1/src/shared/constants.ts#L41-L42

Reply:
I missed that, will update